### PR TITLE
TASK-45348: Wallet view

### PR DESCRIPTION
**Pull request by GouadriaHanen**

### CODE REVIEW

File: wallet-webapps-common/src/main/webapp/css/main.less [hunk 1]
```diff
@@ -593,10 +613,11 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
   }
   .transactionHistoryChart {
     height: 450px;
-    width: 100%;
+    width: 95%;
     max-height: 450px;
     padding: 10px 8px 10px 10px;
     box-sizing: border-box;
+    margin-left: 5% ;
```
Comment (ahamdi) [inline wallet-webapps-common/src/main/webapp/css/main.less]: take care of RTLisation

File: wallet-webapps-common/src/main/webapp/css/main.less [hunk 2]
```diff
@@ -767,3 +773,90 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
...
+    .summaryButtons {
+      width: 100%;
+      .walletSummaryAction{
+        width: 50%
+      }
+      .walletSummaryBalance{
+        display: flex;
+        width: 100%;
+        height: 40px;
+        margin-left : 1%;
```
Comment (ahamdi) [inline wallet-webapps-common/src/main/webapp/css/main.less]: Take care of RTLisation

**Review by ahamdi — CHANGES_REQUESTED** [review]

File: wallet-webapps-common/src/main/webapp/css/main.less (hunk 1 above)
Comment (GouadriaHanen) [reply wallet-webapps-common/src/main/webapp/css/main.less]: done!

File: wallet-webapps-common/src/main/webapp/css/main.less (hunk 2 above)
Comment (GouadriaHanen) [reply wallet-webapps-common/src/main/webapp/css/main.less]: done!

**Review by ahamdi — APPROVED** [review]

**Pull request merged**